qcodes/instrument/ip_to_visa.py

```diff
@@ -94,7 +94,7 @@ def close(self) -> None:
         if hasattr(self, 'connection') and hasattr(self.connection, 'close'):
             self.connection.close()
 
-        strip_attrs(self, whitelist=['name'])
```

Comment:
strip attrs is not gonna strip the "name" property, right? hence thanks to not stripping _short_name the name property will still work after close()?

Reply:
Yes exactly. Before if you tried getting the name property it would complain that there is no property name which is confusing. Now you can get name of a closed instrument (but nothing else)